libs/Microsoft.MixedReality.WebRTC.Unity/Assets/Microsoft.MixedReality.WebRTC.Unity/Scripts/Signaling/NodeDssSignaler.cs

```diff
@@ -248,7 +248,9 @@ private IEnumerator CO_GetAndProcessFromServer()
                             Debug.Log("Unknown message: " + msg.MessageType + ": " + msg.Data);
                             break;
                     }
-                }
+
+					timeSincePollMs = PollTimeMs - 1f; //fast forward next request

```

Comment:
This should be `timeSincePollMs = PollTimeMs + 1f` to ensure the early-out check in `Update()` will always fail, no? Otherwise you will still waste 1 frame in `Update()` to add the frame time, and only after that will the early-out check fail on next frame and the polling execute.

Also can you please fix the tab => space?

Reply:
Yes, ideed.